packages/components/src/Button/ButtonTransparent.tsx

```diff
@@ -25,13 +25,40 @@
  */
 
 import styled from 'styled-components'
+import { variant } from 'styled-system'
 import { ButtonBase } from './ButtonBase'
 
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+export const transparentButtonPadding = variant({
```

Comment:
Super minor but this shouldn't be exported unless it's used elsewhere.

```suggestion
const transparentButtonPadding = variant({
```